docs/key-manager/server-key-manager.md

```diff
@@ -0,0 +1,75 @@
+---
+id: server-key-manager title: Server Key Manager
+---
+
+The [Application Server](../applications/server.md) supports a Neutrino node and wallet. For its wallet, it uses
+a `BIP39KeyManager` for handling its underlying keys and doing cryptographic functions like signing transactions.
+
+### Backups
+
+All seeds are stored in the `seeds` folder inside your bitcoin-s directory, the default location is in `~/.bitcoin-s`.
+The easiest way to back up your seeds is to simply copy the `seeds` folder to your secure back up location.
+
+Do note that if you have a key manager passphrase or a BIP 39 passphrase set for a given seed, you will need these to be
+able to recover the seed.
+
+### Using multiple seeds
+
+You can change which wallet and key manager you are using by setting the `bitcoin-s.wallet.walletName` config
```

Comment:
maybe make it more concrete by adding an example? For instance here is my seeds directory 

> tree ~/.bitcoin-s/seeds
/home/suredbits/.bitcoin-s/seeds
├── encrypted-bitcoin-s-seed.json
├── test-wallet-encrypted-bitcoin-s-seed.json
└── testwallet-encrypted-bitcoin-s-seed.json

0 directories, 3 files